test

```diff
@@ -89,15 +98,35 @@ function grpcproxy_pass {
 }
 
 function release_pass {
+	rm -f ./bin/etcd-last-release
 	# to grab latest patch release; bump this up for every minor release
 	UPGRADE_VER=$(git tag -l --sort=-version:refname "v3.0.*" | head -1)
 	if [ -n "$MANUAL_VER" ]; then
 		# in case, we need to test against different version
 		UPGRADE_VER=$MANUAL_VER
 	fi
-	echo "Downloading" etcd $UPGRADE_VER
-	curl -L https://github.com/coreos/etcd/releases/download/$UPGRADE_VER/etcd-$UPGRADE_VER-linux-amd64.tar.gz -o /tmp/etcd-$UPGRADE_VER-linux-amd64.tar.gz
-	tar xzvf /tmp/etcd-$UPGRADE_VER-linux-amd64.tar.gz -C /tmp/ --strip-components=1
+
+	local file="etcd-$UPGRADE_VER-linux-$MACHINE_ARCH.tar.gz"
+	echo "Downloading $file"
+
+	set +e
```

Comment:
A curl return of 22 (file not found) is success, so we need to check for that, and then in that case just return and not run tar.

If we keep ```set e`` on, then curl's return of 22 will result in test failure.